src/browser-realm.ts

```diff
@@ -19,6 +19,40 @@ const rawEventTargetProto = ReflectGetPrototypeOf(rawWindowPropertiesProto);
 export default function createSecureEnvironment(distortionMap?: Map<SecureProxyTarget, SecureProxyTarget>): (sourceText: string) => void {
     // @ts-ignore document global ref - in browsers
     const iframe = document.createElement('iframe');
+
+    // A comprehensive list of policy feature directives can be found at
+    // https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Feature-Policy#Directives
+    // Directives not currently supported by Chrome are commented out because
+    // Chrome logs warnings to the developer console.
+    iframe.setAttribute('allow', 
```

Comment:
I wonder if this is really needed? lets say we have an iframe, and we detached that iframe, can I access any of these services? I think is either this or the detaching mechanism, both seems excessive.

Reply:
@jdalton what is your take on this?